drivers/ieee802154/Kconfig

```diff
@@ -39,6 +39,13 @@ config IEEE802154_RDEV
 	help
 	  PHY is a ranging-capable device (RDEV)
 
+config IEEE802154_VENDOR_OUI
+	int "Vendor Organizationally Unique Identifier"
+	default 0
+	help
+	  Custom vendor OUI for OpenThread,
```

Comment:
Can we stop referring to OpenThread? It's generic 802.15.4 driver configuration.

Reply:
Sure I forget to change it moving from somewhere else. Good point, done.